problems/objects.js

```diff
@@ -61,7 +78,11 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (appleCountByName) => {
+  // Object.values(appleCountByName)
+  let sum = appleCountByName.Adam + appleCountByName.Eve;
```

Comment:
Can just return, we don't need to create extra variables if we're not using it later. Still works though :-)